src/sentry/tasks/embeddings_grouping/backfill_seer_grouping_records_for_project.py

```diff
@@ -63,28 +62,20 @@ def backfill_seer_grouping_records_for_project(
     Pass in last_processed_group_id = None if calling for the first time. This function will spawn
     child tasks that will pass the last_processed_group_id
     """
+    if options.get("seer.similarity-backfill-killswitch.enabled") or killswitch_enabled(
```

Comment:
Moving the kill switch to the top of the function.

Reply:
Doesn't this mean we're not checking the project killswitch?